chapters/hpmor-chapter-000.tex

```diff
@@ -1,4 +1,4 @@
-\chapter*{Preface}
+\chapter*{Preface by Eliezer Yudkowsky}
```

Comment:
The Preface is by the author unless otherwise stated, so no need to re-state the name.

Reply:
thanks for clarification, fixed